features/step_definitions/web_steps.rb

```diff
@@ -56,16 +56,16 @@
 end
 
 When /I repeatedly answer questions/ do
-  sleep 3
+  sleep 2
```

Comment:
This is super minor, but maybe add a comment here briefly explaining why we need a sleep here, so future teams don't get confused

Reply:
```suggestion
  sleep 2 # Sleeps are necessary here to allow for CSS transition times
```